tests/test_enum.py

```diff
@@ -238,13 +238,13 @@ def test_duplicate_enum_name():
 
 def test_char_underlying_enum():  # Issue #1331/PR #1334:
     assert type(m.ScopedCharEnum.Positive.__int__()) is int
-    assert int(m.ScopedChar16Enum.Zero) == 0  # int() call should successfully return
+    assert int(m.ScopedChar16Enum.Zero) == 0
     assert hash(m.ScopedChar32Enum.Positive) == 1
-    assert m.ScopedCharEnum.Positive.__getstate__() == 1  # return type is long in py2.x
+    assert m.ScopedCharEnum.Positive.__getstate__() == 1
```

Comment:
Yes, the intent of this test is to exercise `__getstate__()` call (along with `__hash__()` call). The intent of this comment was because `__getstate__()` and `__hash__()` returns a `long` type in python 2.x, so 
```
assert type(m.ScopedCharEnum.Positive.__getstate__()) is int
```
will fail in python 2.x platforms.  So I tested them by comparing with 1.

Reply:
Aha! :-)
I changed this, using `if env.PY2`, so that we get what you originally had in mind for Python 3.

The `if env.PY2` branch will get purged when we clean out Python 2 support in a few months.

Unfortunately our pre-commit checks (black,  flake8) get upset about `type(...) is long`. To not make things overly complicated for Python 2 I just kept your `== 1` test for Python 2.